src/utils/dateHelpers.ts

```diff
@@ -1,28 +1,28 @@
+import * as moment from 'moment'
+
 /**
  * Checks if a date is before or at the current time.
  *
- * @param {string} date - Current date, in a parsable string format
+ * @param {string} d an ISO 8601 compliant date string
  *   e.g 2018-07-06T10:20:09.961Z
  * @returns {boolean} true if the date specified by the paramter
  *   is before the time of execution of this function.
  */
 export const beforeNow = (d: string): boolean => {
-  const date = new Date(d)
-  const now = new Date()
-  return date <= now
+  return moment(d).isBefore(moment())
```

Comment:
I don't think we were aware of this earlier, but backend sends us overviews even for all assessments, even those that are not yet open.

For these 'pending' assessment overviews, we need to place them in their own collapsible category, and hide their 'attempt' buttons (but we should not hide the attempt buttons for staff and admins https://github.com/source-academy/cadet/pull/195). Can you implement this change in this PR?